common/aws/s3/fragment.go

```diff
@@ -0,0 +1,128 @@
+package s3
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// GetFragmentCount returns the number of fragments that a file of the given size will be broken into.
+func GetFragmentCount(fileSize int, fragmentSize int) int {
```

Comment:
I feel like if these Fragment related functions are going to be in the s3 package, then they should all be private (lowerCase)? Can be tested by moving the `fragment_test.go` function inside the s3 package.

Reply:
Done. Easy enough to make them public again later if we discover a need.